veg2hab/vegkartering.py

```diff
@@ -453,9 +454,9 @@ def build_aggregate_habtype_field(row: gpd.GeoSeries) -> str:
         assert (
             len(habitatkeuzes) == 1
         ), "Bij KeuzeStatus GEEN_OPGEGEVEN_VEGTYPE mag er maar 1 habitatkeuze zijn"
-        assert (
-            vegtypeinfos == [VegTypeInfo(percentage=100, SBB=[], VvN=[])],
-        ), "Bij KeuzeStatus GEEN_OPGEGEVEN_VEGTYPE moet er een leeg 100% VegTypeInfo zijn"
```

Comment:
Er stond hier een comma teveel, dus er werd per ongeluk gechecked of de tuple niet leeg was, wat natuurlijk altijd waar is bij een tuple van lengte 1.